core/controllers/base.py

```diff
@@ -328,7 +329,8 @@ def render_template(
                 rights_manager.ACTIVITY_STATUS_PUBLIC),
             'ACTIVITY_STATUS_PUBLICIZED': (
                 rights_manager.ACTIVITY_STATUS_PUBLICIZED),
-            'FULL_URL': '%s://%s/%s' % (scheme, netloc, path),
+            # path starts with a forward slash.
```

Comment:
The 'path' variable starts with a forward slash.

(Took me a while to realize you were actually referring to the 'path' variable, instead of the general path.)